changelog/3637.feature.2.rst

```diff
@@ -0,0 +1 @@
+Printing `sunpy.net.attr.Instrument` or other attributes will print all the values that exist under the name space.
```

Comment:
I am confused by this, under what namespace? Maybe an example would be helpful

Reply:
Reworded.